lib-support.h

```diff
@@ -17,6 +17,11 @@
 #include "arch/x86_64.h"
 #elif defined(__arm__)
 #include "arch/arm.h"
+#elif defined(__aarch64__)
+#if __GNUC__ < 7
```

Comment:
Isn't the problem that `.data.rel.` section mismatches with expected PIC address resulting from GNU ld rather than gcc? If so, it is useless to verify GCC versions.

Reply:
Yes, right.
ld version need >= 2.28

Reply:
I'll try other ways to verify LD version.

Reply:
Validating `ld` versions in `Makefile` should be fine.